Docs/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging
+
+The content of this guide will help you to configure your PHPStorm in order to debug the PIM.
+The following doc is about Linux only and is not tested on other OS.
+
+## Container config
+
+*Xdebug* is deactivated by default. If you want to activate it, you need to set the following environment variables,
+ in this case we use a part of the default configuration of the Akeneo PIM 2.0 which use fpm and httpd but this is the same spirit for others versions:
```

Comment:
Better to make several sentences (always short sentences in english) => put a period after `variables` instead of a comma.